api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -90,6 +90,16 @@ message ExtAuthz {
   //
   // If this field is not specified, the filter will be enabled for all requests.
   api.v2.core.RuntimeFractionalPercent filter_enabled = 9;
+
+  // Specifies the :ref:`labels<envoy_api_field_service.auth.v2.AttributeContext.Peer.labels>`
+  // to send to the external service.
+  //
+  // The available labels are:
+  //
+  // `peer-certificate`: The peer X.509 certificate, in URL- and PEM-encoded format.
+  //
+  // By default no label is sent to the external service.
```

Comment:
This is much better. Do you think we could do a static doc with this list and avoid repeating here?

I have one other question; if these things are well defined, why don't we just a bunch of bool fields to represent?

Reply:
Using bool fields is more explicit.  And if we estimate that there will not be many labels (this is the first) then it might not become too unwieldy to use one field per label.

I've pushed a change using a bool field.  LMK what you think, @htuch.